northstar/src/runtime/island/fs.rs

```diff
@@ -39,23 +39,23 @@ pub(super) struct Mount {
     pub fstype: Option<&'static str>,
     pub flags: MsFlags,
     pub data: Option<String>,
+    pub err_str: String,
 }
 
+// TODO: The container could be malformed and the mountpoint might be missing.
+// This is not a fault of the RT so don't expect it.
```

Comment:
The mount points are in the the roofs and under the responsibility of the one that creates a NPK. The runtime cannot create the mount points because of the read onliness of squashfs. This means that malicious container might lack mount points.